desktop/core/src/desktop/auth/backend.py

```diff
@@ -579,7 +579,10 @@ def authenticate(self, request=None, username=None, password=None, server=None):
     try:
       allowed_group = self.check_ldap_access_groups(server, username)
       if allowed_group:
-        user = self._backend.authenticate(username=username, password=password)
+        if sys.version_info[0] > 2:
+          user = self._backend.authenticate(request, username=username, password=password)
+        else:
+          user = self._backend.authenticate(username=username, password=password)
```

Comment:
Yes, @romainr unit test was not passing with 'request' argument for Django 1. Thanks a lot.